cli/cmd/user/list.go

```diff
@@ -8,6 +8,12 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	UserTokenPrefix   = "usr" // User token prefix
+	InviteTokenPrefix = "inv" // Invite token prefix
```

Comment:
Should not be exported, they are and should only be accessed inside the `ListCmd` command. (These are not global prefixes, they are internal to the command to disambiguate user input.)